File: Data Structures & Algorithms/non-overlapping-intervals/submission-0.java
class Solution {
    public int eraseOverlapIntervals(int[][] intervals) {
         Arrays.sort(intervals,(a,b)->Integer.compare(a[1],b[1]));
        int count=0;
        List<int[]>ans=new ArrayList<>();
        ans.add(intervals[0]);
        for(int i=1;i<intervals.length;i++){
            int []curr=intervals[i];
            int []last=ans.get(ans.size()-1);
            if(curr[0]<last[1]){
                count++;
            }
            else{
                ans.add(intervals[i]);
            }
        } 
        return count;
    }
}

File: Data Structures & Algorithms/climbing-stairs/submission-0.cpp
class Solution {
public:
    int climbStairs(int n) {
        int prev1=0;
        int prev=1;
        int curr;
        for(int i=1;i<=n;i++){
            curr=prev1+prev;
            prev1=prev;
            prev=curr;
        }
        return prev;
    }
};
